index.html

```diff
@@ -1,12 +1,33 @@
-<h1>Sukonkina Arina</h1>
-<h2>About</h2>
+<!DOCTYPE html>
+<html>
+    <head>
+        <title><b>My profile</b></title>
```

Comment:
The content of the `<title>` tag is not affected by formatting tags, such as `<b>`. That's the title on the tab, not on the page. The tag actually appears as literal text, so you can remove it.
```suggestion
        <title>My profile</title>
```